README.md

```diff
@@ -9,23 +9,21 @@
 
 [Getting Started](https://www.mackelab.org/sbi/tutorial/00_getting_started/) | [Documentation](https://www.mackelab.org/sbi/)
 
-`sbi` is a PyTorch package for simulation-based inference. Simulation-based inference is  
-the process of finding parameters of a simulator from observations.
+`sbi` is a PyTorch package for simulation-based inference. Simulation-based inference is the process of finding parameters of a simulator from observations.
 
 `sbi` takes a Bayesian approach and returns a full posterior distribution
-over the parameters, conditional on the observations. This posterior can be amortized (i.e.
-useful for any observation) or focused (i.e. tailored to a particular observation), with different
-computational trade-offs.
+over the parameters, conditional on the observations. This posterior can be amortized (i.e. once trained, it can be
+applied to any observation) or focused (i.e. tailored to a particular observation), with different computational trade-offs.
 
-`sbi` offers a simple interface for one-line posterior inference.
+`sbi` offers a simple interface for one-line posterior inference:
 
 ```python
 from sbi.inference import infer
 # import your simulator, define your prior over the parameters
-parameter_posterior = infer(simulator, prior, method='SNPE', num_simulations=100)
+parameter_posterior = infer(simulator, prior, method='NPE', num_simulations=100)
```

Comment:
I'm confused about how this can work. Since the class are called SNPE won't this raise an error [here](https://github.com/mackelab/sbi/blob/96d65e8a8e763c651e356df1a8a87e397fda87c2/sbi/inference/base.py#L63)?

Reply:
yes, this does not make sense. thanks for spotting it. My intention was to advertise NPE, but we can just write it in the docs.